pkg/apis/core/v1alpha1/defaults.go

```diff
@@ -325,6 +351,13 @@ func SetDefaults_Shoot(obj *Shoot) {
 	}
 }
 
+func isDockerRuntime(cri *CRI) bool {
+	if cri != nil {
+		return cri.Name == CRINameDocker
+	}
+	return false
+}
+
```

Comment:
All comments in this file also apply for `v1beta1` package.

Reply:
yep done